core-java-modules/core-java-reflection/src/test/java/com/baeldung/reflection/exception/invocationtarget/InvocationTargetUnitTest.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.reflection.exception.invocationtarget;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import org.junit.jupiter.api.Test;
+
+public class InvocationTargetUnitTest {
+
+    @Test
+    public void whenCallingMethodThrowsException_thenAssertTrue() throws Exception {
+        InvocationTargetExample targetExample = new InvocationTargetExample();
+        Method method = InvocationTargetExample.class.getMethod("divideByZeroExample");
+        Exception exception = assertThrows(InvocationTargetException.class, () -> method.invoke(targetExample));
```

Comment:
add a blank line before and after this line (given, when, then)

Reply:
I've added the blank lines before when and then

Reply:
I've added two blank lines because here i am asserting InvocationTargetException and its cause